fdbserver/storageserver.actor.cpp

```diff
@@ -2683,7 +2729,7 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 				// wait( data->fetchKeysStorageWriteLock.take() );
 				// state FlowLock::Releaser holdingFKSWL( data->fetchKeysStorageWriteLock );
 
-				// Write this_block to storage
+				// Write this_block directly to storage, bypassing update() which updates in the memory.
```

Comment:
Maybe specify that `update()` will write to MVCC. To memory could be ambiguous in some cases.

Reply:
Fixed.